src/main/java/seedu/address/model/affiliation/exceptions/SameRoleAffiliationException.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.model.affiliation.exceptions;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.Person;
+
+
+/**
+ * Throws a SameRoleAffiliationException exception which is a CommmandException
+ * when the affiliation person's role is the same as the role of the person adding affiliation.
+ */
+public class SameRoleAffiliationException extends CommandException {
+
+    /**
+     * Constructs a SameRoleAffiliationException.
+     * @param personNotAdded The person that fails to add in affiliation.
+     */
+    public SameRoleAffiliationException(Person personNotAdded) {
+        super(personNotAdded.getName().toString() + " cannot be added as it has the same role "
```

Comment:
Is possible to adjust this to "... cannot be added <ins>as an affiliation</ins> as it has the same role <ins>of</ins> ..." with full stop?

Reply:
Thanks for pointing it out. Will change it in the next commit.